### PR TITLE
Rename `AppendEncoded()` to `AppendHtml()`

### PR DESCRIPTION
**Pull request by dougbu**
- aspnet/Mvc#3225, 2 of 3

### CODE REVIEW

**Comment by dougbu** [comment]
@kichalla

**Comment by Eilon** [comment]
:shipit:

**Comment by NTaylorMullen** [comment]
:shipit: once all PRs for other repos are good to go

**Comment by dougbu** [comment]
@NTaylorMullen  CI builds are failing because HttpAbstractions changes aren't in. This is ready to go and I've tested things locally. Main new change here is `SetContentEncoded()` -> `SetHtmlContent()`.

File: test/Microsoft.AspNet.Razor.Runtime.Test/TagHelpers/DefaultTagHelperContentTest.cs [hunk 1]
```diff
@@ -475,11 +475,11 @@ public void Append_WrittenAsEncoded()
         }
 
         [Fact]
-        public void AppendEncoded_DoesNotGetEncoded()
+        public void AppendHtml_DoesNotGetEncoded()
```
Comment (NTaylorMullen) [inline test/Microsoft.AspNet.Razor.Runtime.Test/TagHelpers/DefaultTagHelperContentTest.cs]: Shocked we don't have a `SetHtmlContent` equivalent. Could you add a single test while you're here real quick?

Comment (dougbu) [inline test/Microsoft.AspNet.Razor.Runtime.Test/TagHelpers/DefaultTagHelperContentTest.cs]: :ok:

**Comment by dougbu** [comment]
Updated FYI

**Comment by NTaylorMullen** [comment]
Beautiful gogogogo :smile:

**Comment by dougbu** [comment]
117bbe7

**Pull request closed without merging**